packages/block-library/src/gallery/shared.js

```diff
@@ -19,5 +24,7 @@ export const pickRelevantMediaFiles = ( image, sizeSlug = 'large' ) => {
 	if ( fullUrl ) {
 		imageProps.fullUrl = fullUrl;
 	}
+	imageProps.alt =
+		imageProps.alt !== '' ? imageProps.alt : __( 'Image Gallery Image' );
```

Comment:
Should this be sentence case vs Title Case?

Reply:
fixed.